<launch>
  <node pkg="navigation_cli" type="velocity_smoother" name="velocity_smoother_node" output="screen">
  <param name="raw_cmd_vel_topic" value="/mobile_base/commands/velocity" />
  <param name="smooth_cmd_vel_topic" value="/mobile_base/commands/velocity/smooth" />
  <param name="odom_topic" value="" />
  <param name="frequency" value="100" />
  <param name="v_max_vel" value="0.3" />
  <param name="v_min_vel" value="-0.05" />
  <param name="v_max_acc" value="2" />
  <param name="v_min_acc" value="-2" />
  <param name="v_jeck" value="2.0" />
  <param name="a_max_vel" value="0.3" />
  <param name="a_min_vel" value="-0.3" />
  <param name="a_max_acc" value="3" />
  <param name="a_min_acc" value="-3" />
  <param name="a_jeck" value="3.0" />
  </node>
</launch>
